media_driver/linux/gen12/ddi/media_sysinfo_g12.cpp

```diff
@@ -175,8 +175,23 @@ static struct GfxDeviceInfo tgllpGt2Info = {
 static bool tgllpGt2Device9a09 = DeviceInfoFactory<GfxDeviceInfo>::
```

Comment:
Can we remove 0x9a09 here for it is not used?

Reply:
It was here before. I prefer someone else would submit another PR for cleanup if it is really needed.

Reply:
Thank you for the  clean up of 0x9109 (8a8b8ae263a04cb5dd384d4625215cf296d2cc59). I've rebased my PR on top.